apps/accounts/forms/update.py

```diff
@@ -146,7 +147,10 @@ def __init__(self, instance, *args, **kwargs):
         """Initialization for TimeZoneForm.
         """
         user_profile = instance
+        timezone = [user_profile.timezone for tz in US_TIMEZONE_CHOICES if user_profile.timezone in tz]
+        user_profile.timezone = timezone[0] if timezone else htk_setting('HTK_DEFAULT_TIMEZONE', 'UTC')
         super(TimeZoneForm, self).__init__(user_profile, *args, **kwargs)
+        self.fields['timezone'].choices = [(x, y,) for x, y in US_TIMEZONE_CHOICES]
```

Comment:
Don't hardcode the timezone choices here

Instead, pass it in `**kwargs`

```
timezone_choices = kwargs.get('timezone_choices', None)
if timezone_choices:
    self.fields['timezone'].choices = [('', '', ), ] + timezone_choices
```

(include a blank choice)

Reply:
Done